templates/download/cloud/index.html

```diff
@@ -5,14 +5,14 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1kp7QQXst1EbCjZjzWZP3YZwfmG0EIXLfQoRUVaMEZng/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip--image u-no-background--small is-deep is-bordered" style="background-image: url('https://assets.ubuntu.com/v1/1d2ab6ba-suru-background.png'); background-size: 100% 100%;">
+<section class="p-strip--suru-topped is-bordered">
   <div class="row u-equal-height">
     <div class="col-8">
       <h1>Optimised Ubuntu on&nbsp;public&nbsp;clouds</h1>
       <p>Ubuntu builds optimised and certified server images for partners like Amazon AWS, Microsoft Azure, Google Cloud Platform, Oracle, Rackspace and IBM Cloud. In addition, Canonical offers Ubuntu Advantage, enterprise-grade commercial support, on these images.</p>
     </div>
     <div class="col-4 u-align--center u-vertically-center u-hide--small">
-      <img src="https://assets.ubuntu.com/v1/1c786630-image-cloud.svg" width="323" alt="Ubuntu cloud">
+      <img src="https://assets.ubuntu.com/v1/a9580e02-ubuntu-cloud-medium.svg" width="350" alt="Ubuntu cloud">
```

Comment:
Could use the image module here too:
```suggestion
          {{
            image(
              url="https://assets.ubuntu.com/v1/a9580e02-ubuntu-cloud-medium.svg",
              alt="",
              width="189",
              height="319",
              hi_def=True,
            ) | safe
          }}
```

Reply:
Thanks, updated.